core/src/main/java/nl/nn/adapterframework/pipes/AbstractPipe.java

```diff
@@ -403,7 +403,7 @@ public IAdapter getAdapter() {
 
 	/**
 	 * This ClassLoader is set upon creation of the pipe, used to retrieve resources configured by the Ibis application.
-	 * @return returns the ClassLoader created by the {@link ClassLoaderManager ClassLoaderManager}.
+	 * @return returns the ClassLoader created by the {@link nl.nn.adapterframework.configuration.ClassLoaderManager ClassLoaderManager}.
```

Comment:
Deze verandering hoeft niet. ClassLoaderManager is bij de imports opgenomen.

Reply:
Gedaan.